packages/paste-core/components/tabs/src/Tab.tsx

```diff
@@ -89,7 +90,7 @@ const getTabBoxStyles = (orientation: Orientation, variant: Variants): BoxStyleP
   }
 };
 
-export interface TabProps {
+export interface TabProps extends BoxElementProps {
```

Comment:
Is `BoxElementProps` too much for `TabProps`? Since `BoxElementProps` with various html elements props that a Tab may not ever need, i.e. image props like `alt` and `src`.

Reply:
Fair